commerce_coordinator/apps/rollout/pipeline.py

```diff
@@ -28,7 +31,18 @@ def run_filter(self, request):
         sku = request.query_params.get('sku', '').strip()
         course_run = request.query_params.get('course_run_key', '').strip()
 
-        if is_redirect_to_commercetools_enabled_for_user(request) and course_run:
+        ct_api_client = CommercetoolsAPIClient()
+        # breakpoint()
+        try:
+            commercetools_available_course =  ct_api_client.get_product_variant_by_course_run(course_run)
+        except HTTPError as exc:
+            logger.exception(
+                '[get_product_variant_by_course_run] Failed to get Commercetools course for course_run: %s', course_run
+            )
+            raise APIException("Error while fetching course variant from Commercetools") from exc
```

Comment:
Can we use the legacy system as a fallback option whenever CT SDK throws an error or the product is not found?

```suggestion
        commercetools_available_course = None
        if course_run:
            try:
                commercetools_available_course =  ct_api_client.get_product_variant_by_course_run(course_run)
            except HTTPError as exc:
                logger.exception(
                    '[get_product_variant_by_course_run] Failed to get Commercetools course for course_run: %s', course_run
                )
```